test/common/nighthawk_service_client_test.cc

```diff
@@ -0,0 +1,154 @@
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/client/options.pb.h"
+#include "api/client/service.grpc.pb.h"
+#include "api/client/service_mock.grpc.pb.h"
+
+#include "common/nighthawk_service_client_impl.h"
+
+#include "grpcpp/test/mock_stream.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+
+using ::Envoy::Protobuf::util::MessageDifferencer;
+using ::nighthawk::client::CommandLineOptions;
+using ::nighthawk::client::ExecutionRequest;
+using ::nighthawk::client::ExecutionResponse;
+using ::testing::_;
+using ::testing::DoAll;
+using ::testing::HasSubstr;
+using ::testing::Return;
+using ::testing::SaveArg;
+using ::testing::SetArgPointee;
+
+TEST(PerformNighthawkBenchmark, UsesSpecifiedCommandLineOptions) {
+  const int kExpectedRps = 456;
+  ExecutionRequest request;
+  nighthawk::client::MockNighthawkServiceStub mock_nighthawk_service_stub;
+  EXPECT_CALL(mock_nighthawk_service_stub, ExecutionStreamRaw)
```

Comment:
Two small asks, but on each of these tests.

1. Can you add a linebreak between this EXPECT_CALL and the CommandLineOptions below (or in other tests, whatever is following it)?  Just helps the eyes parse this a little as one block.

2. Can we comment at the top of the EXPECT_CALL statement roughly what this block is mocking?  Or if that seems like a bad way to document this, something else that helps clarify for future readers what is happening.

Reply:
Done